docs/multi-stage-query/api.md

```diff
@@ -90,7 +90,7 @@ import json
 import requests
 
 # Make sure you replace `username`, `password`, `your-instance`, and `port` with the values for your deployment.
```

Comment:
Please break up this comment into 2 parts as we are replacing only `your-instance` and `port`. 
The first part can mention `your-instance` and `port`. 
The second part can go above line 105 where we do the post. 

Similar comment everywhere else.